Description: Revert sage commit 5dbc699911095dd03f0141a7c1b7ba1c7056ceef
 This is dependent on a python patch that hasn't landed upstream or in Debian
 yet. It adds a minor feature used nowhere except the doctests, so just revert
 it because we know it will break.
Author: Ximin Luo <infinity0@debian.org>
Bug-Python: https://bugs.python.org/issue27177
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sage/src/sage/rings/integer.pyx
+++ b/sage/src/sage/rings/integer.pyx
@@ -765,16 +765,6 @@
             4
             sage: v[Integer(2):Integer(4)]
             [3, 4]
-
-        See :trac:`20750`::
-
-            sage: import re
-            sage: p = re.compile('(a)b')
-            sage: m = p.match('ab')
-            sage: m.group(Integer(0))
-            'ab'
-            sage: m.group(Integer(1))
-            'a'
         """
         return mpz_get_pyintlong(self.value)
 
